packages/flutter/lib/src/services/platform_views.dart

```diff
@@ -1172,15 +1172,20 @@ class TextureAndroidViewController extends AndroidViewController {
   Future<void> _sendCreateMessage({required Size size, Offset? position}) async {
     assert(!size.isEmpty, 'trying to create $TextureAndroidViewController without setting a valid size.');
 
-    _internals.textureId = await _AndroidViewControllerInternals.sendCreateMessage(
+    final dynamic response = await _AndroidViewControllerInternals.sendCreateMessage(
```

Comment:
Why `dynamic` rather than `int?`?

Reply:
Consistency: I copied code from elsewhere in this file that uses the exact same control flow around a call to sendCreateMessage.